src/celengine/render.cpp

```diff
@@ -2966,13 +2966,13 @@ void Renderer::renderPlanet(Body& body,
         }
     }
 
-    if (body.isVisibleAsPoint())
+    if (body.isVisibleAsPoint() && body.getSurface().color.toVector3().maxCoeff() > 0.0f)
```

Comment:
```suggestion
    if (body.isVisibleAsPoint())
    {
        if (float maxCoeff = body.getSurface().color.toVector3().maxCoeff(); maxCoeff > 0.0f)
        {
```

Reply:
That one is meant to avoid causing the program to divide by 0, but maybe there is a better way to do this.